client/src/components/Layout/General.js

```diff
@@ -1,12 +1,33 @@
 import React from 'react';
 import PropTypes from 'prop-types';
 
+import Icon from '../Icon';
+
+import SideBar from './SideBar';
+import Menu from './Menu';
+
+import { Layout, Header, AntdFooter, AntdContent } from './style';
```

Comment:
```suggestion
import * as S from './style';
```
so you can distinguish between styled-components and common/imported components

Reply:
done